MAINTENANCE.md

```diff
@@ -89,6 +89,14 @@ strcspn requires the data must end with a NUL, so if we cannot guarantee that th
 
 https://github.com/r-lib/vroom/issues/357 tracks this issue
 
-## External "standards"
+### External "standards"
 
 https://datatracker.ietf.org/doc/html/rfc4180
+
+## Ideas we are not pursuing
+
+While these ideas are not bad, the following projects are not currently being pursued until we have more maintainer bandwidth.
+
+### Replicating functions from readr
+
+If there is a good, working solution from readr, we are not currently planning to replicate that behavior in vroom.
```

Comment:
This doesn't feel like a specific idea or project that we aren't pursuing.

Perhaps it would be better re-phrased as a default policy that informs current decision-making and stick it in the intro to this section.
I think it really speaks to how we will prioritize things.